kids/lib/components/notification_prompt.dart

```diff
@@ -67,14 +67,24 @@ class NotificationPrompt extends HookConsumerWidget {
                       return;
                     }
 
+                    final anonymousId = await ref.read(analyticsProvider).getAnonymousId();
+
                     switch (permissions.authorizationStatus) {
                       case AuthorizationStatus.authorized:
                         debugPrint('notification permission status: accepted');
                         analytics.notificationPromptAccepted(NotificationPromptAcceptedEvent());
+                        analytics.notificationsStatus(NotificationsStatusEvent(
+                          recipientId: anonymousId,
+                          enabled: true,
+                        ));
                         settings.setNotificationsEnabled(true);
                       case AuthorizationStatus.denied:
                         debugPrint('notification permission status: denied');
                         analytics.notificationPromptDenied(NotificationPromptDeniedEvent());
+                        analytics.notificationsStatus(NotificationsStatusEvent(
+                          recipientId: anonymousId,
+                          enabled: false,
+                        ));
```

Comment:
This is a bit messy, but I'd like to get it out of the door before I do any more work